### PR TITLE
Prometheus RemoteWrite Docs

### PR DESCRIPTION
**Pull request by mcbenjemaa**
Towards: https://github.com/giantswarm/roadmap/issues/1238

### Please remember to

- [X] Give the PR a meaningful title -- it will be shown in the release notes
- [ ] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/main/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

**Comment by QuentinBisson** [comment]
@giantswarm/team-atlas can you take a quick look before @mcbenjemaa comes back ?

File: src/content/advanced/prometheus-remotewrite/index.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+## Introduction to remote write is Prometheus RemoteWrite
+
+Prometheus allows users to replicate it's data into 3rd party systems like Cortex or even another Prometheus using its [remote APIs](https://prometheus.io/blog/2019/10/10/remote-read-meets-streaming/#remote-apis).
+To enable this behavior, Prometheus needs to be configured using [remote_write config](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write) to define which endpoints to send data to.
+
+## What is the Giant Swarm Prometheus RemoteWrite feature
+
+The Prometheus `RemoteWrite` feature that allows you to configure remote write targets in the Giant Swarm managed prometheus set up on your management cluster.
+That way, you can replicate and send metrics to your own 3rd party system and use the data as you see fit for the component running on the management clusters such as Flux
```
Comment (TheoBrigitte) [inline src/content/advanced/prometheus-remotewrite/index.md:30]: I would remove the Flux example here, it does not give a concrete enough example, and its confusing to talk about management cluster concept here IMO.

```suggestion
That way, you can replicate and send metrics to your own 3rd party system and use the data as you see fit.
```

File: src/content/advanced/prometheus-remotewrite/index.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+Giant Swarm provides and uses a `RemoteWrite` Custom Resource Definition that you can use to configure `remote_write` targets.
+
+For example, this "RemoteWrite" Custom Resource indicates that we configure the remote write feature for all clusters (`clusterSelector: {}`) to send the data to grafana cloud (`url: https://prometheus-us-central1.grafana.net/api/prom/push`)
```
Comment (TheoBrigitte) [inline src/content/advanced/prometheus-remotewrite/index.md:34]:
```suggestion
For example, this `RemoteWrite` Custom Resource indicates that we configure the remote write feature for all clusters (`clusterSelector: {}`) to send the data to grafana cloud (`url: https://prometheus-us-central1.grafana.net/api/prom/push`)
```

File: src/content/advanced/prometheus-remotewrite/index.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+```yaml
+apiVersion: monitoring.giantswarm.io/v1alpha1
+kind: RemoteWrite
+metadata:
+  name: grafana-cloud
+  namespace: monitoring
+spec:
+  ## Defines the cluster to configure prometheus remote write for
```
[the remark this answers is not in the document]
Comment (TheoBrigitte) [reply src/content/advanced/prometheus-remotewrite/index.md:43]: Maybe also mention that namespace are not filtered here.

**Review by TheoBrigitte — COMMENTED** [review]
Nice documentation.

Just few minor remarks

File: src/content/advanced/prometheus-remotewrite/index.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+aliases:
+  - /advanced/remotewrite/
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+last_review_date: 2022-07-19
+---
+
+In this document you will learn how to use the Prometheus RemoteWrite feature provided out of the box by Giant Swarm.
+
+## Introduction to remote write is Prometheus RemoteWrite
```
Comment (hervenicol) [inline src/content/advanced/prometheus-remotewrite/index.md:22]:
```suggestion
## Introduction to Prometheus RemoteWrite
```

File: src/content/advanced/prometheus-remotewrite/index.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+Prometheus allows users to replicate it's data into 3rd party systems like Cortex or even another Prometheus using its [remote APIs](https://prometheus.io/blog/2019/10/10/remote-read-meets-streaming/#remote-apis).
```
Comment (hervenicol) [inline src/content/advanced/prometheus-remotewrite/index.md:24]:
```suggestion
Prometheus allows users to replicate its data into 3rd party systems like Grafana Cloud or even another Prometheus using its [remote APIs](https://prometheus.io/blog/2019/10/10/remote-read-meets-streaming/#remote-apis).
```

File: src/content/advanced/prometheus-remotewrite/index.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+To enable this behavior, Prometheus needs to be configured using [remote_write config](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write) to define which endpoints to send data to.
+
+## What is the Giant Swarm Prometheus RemoteWrite feature
+
+The Prometheus `RemoteWrite` feature that allows you to configure remote write targets in the Giant Swarm managed prometheus set up on your management cluster.
```
Comment (hervenicol) [inline src/content/advanced/prometheus-remotewrite/index.md:29]:
```suggestion
The Prometheus `RemoteWrite` feature allows you to configure remote write targets in the Giant Swarm managed prometheus running on your management cluster.
```

**Review by hervenicol — COMMENTED** [review]
Small fixes

File: src/content/advanced/prometheus-remotewrite/index.md (hunk 1 above)
Comment (QuentinBisson) [reply src/content/advanced/prometheus-remotewrite/index.md:30]: Well the feature is only available on the management clusters and was made for flux originally :D but sure

**Review by TheoBrigitte — APPROVED** [review]
LGTM

**Comment by QuentinBisson** [comment]
Closes https://github.com/giantswarm/roadmap/issues/1238

**Pull request merged**